cmd/analyse_test.go

```diff
@@ -0,0 +1,58 @@
+package cmd
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/damianoneill/nc-hammer/result"
+)
+
+var ts1 = result.NetconfResult{0, 0, "10.0.0.1", "get", 0, "error", 0}
+var ts2 = result.NetconfResult{0, 0, "10.0.0.2", "get", 0, "", 0}
+var ts3 = result.NetconfResult{0, 0, "10.0.0.3 ", "get", 0, "", 0}
+var ts4 = result.NetconfResult{0, 0, "10.0.0.3 ", "edit-config", 0, "", 0}
```

Comment:
you have examples for get and edit-config, show an example for get-config the other valid option and possible include a test for an invalid option such as kill-session.